xml/slurm.xml

```diff
@@ -15,9 +15,9 @@
   <abstract>
    <para>
     <emphasis>Slurm</emphasis> is a workload manager for managing compute jobs
-    on high-performance computing clusters. It can start multiple jobs on a
-    single node or a single job on multiple nodes. Additional components can be
-    used for advanced scheduling and accounting.
+    on high-performance computing clusters. It is able to start multiple jobs on
+    a single node or a single job on multiple nodes. Additional components can
+    be used for advanced scheduling and accounting.
```

Comment:
This looks like a revert of a `s/be able to/can/` style fix, so I would reset to the old version:

```suggestion
    on high-performance computing clusters. It can start multiple jobs on a
    single node or a single job on multiple nodes. Additional components can be
    used for advanced scheduling and accounting.
```